<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmreports</class>
 <widget class="QWidget" name="frmreports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>445</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Choose one report from the list:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Browse to load a report:</string>
       </property>
       <property name="buddy">
        <cstring>lineOpen</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineOpen"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushOpen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Open report</string>
         </property>
         <property name="statusTip">
          <string>Browse files to load a report from non-standard location</string>
         </property>
         <property name="whatsThis">
          <string>Browse files to load a report from non-standard location</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="app_new.qrc">
           <normaloff>:/app_new/folder.png</normaloff>:/app_new/folder.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="lbPixmap">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Report Icon</string>
        </property>
        <property name="statusTip">
         <string>Report Icon</string>
        </property>
        <property name="whatsThis">
         <string>Report Icon</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="app_new.qrc">:/app_new/kpresenter.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>-&gt; TYPE SOMETHING HERE</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Author:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbAuthor">
          <property name="toolTip">
           <string>Report Author</string>
          </property>
          <property name="statusTip">
           <string>Report Author</string>
          </property>
          <property name="whatsThis">
           <string>Report Author</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>-&gt; TYPE SOMETHING HERE</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="buddy">
         <cstring>textDescription</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QTextBrowser" name="textDescription">
        <property name="toolTip">
         <string>Report description</string>
        </property>
        <property name="statusTip">
         <string>Report description</string>
        </property>
        <property name="whatsThis">
         <string>Report description</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="toolTip">
        <string>Close</string>
       </property>
       <property name="statusTip">
        <string>Close</string>
       </property>
       <property name="whatsThis">
        <string>Close</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>textDescription</tabstop>
  <tabstop>lineOpen</tabstop>
  <tabstop>pushOpen</tabstop>
  <tabstop>pushClose</tabstop>
 </tabstops>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>frmreports</receiver>
   <slot>previewItem(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>frmreports</receiver>
   <slot>loadItem(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushOpen</sender>
   <signal>clicked()</signal>
   <receiver>frmreports</receiver>
   <slot>browseFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>previewItem(QListWidgetItem*)</slot>
  <slot>loadItem(QListWidgetItem*)</slot>
  <slot>browseFiles()</slot>
 </slots>
</ui>
